pkg/object/file.go

```diff
@@ -326,6 +324,7 @@ func (d *filestore) ListAll(prefix, marker string) (<-chan Object, error) {
 				owner,
 				group,
 				info.Mode(),
+				info.Mode()&os.ModeSymlink != 0,
```

Comment:
In order to follow symlink, we should return the info of it's target (may also ignore the broken symlink?)